keyboards/handwired/asesmero/6x6/info.json

```diff
@@ -0,0 +1,72 @@
+{
+    "manufacturer": "asesmero",
+    "keyboard_name": "6x6 promicro",
+    "maintainer": "asesmero",
+	"url": "https://github.com/asesmero"
+    "development_board": "promicro",
+	
+    "usb": {
+        "device_version": "0.0.1",
+        "pid": "0x0066",
+        "vid": "0x0282"
```

Comment:
This makes me uncomfortable.  While yes, it's not assigned officially...

Reply:
In general, I prefer VIDs that are much higher in number.  Past 0x4000 is good, and the further past, the better.   For instance, `0x5A30`, which was generated with this command: 
```sh
echo -n 'asesmero' | shasum | cut -c1-4 | tr '[:lower:]' '[:upper:]'
```